index.html

```diff
@@ -14,7 +14,7 @@
 
         <div class="page-heading">How Istanbul works</div>
         <p>
-          Istanbul instruments your ES5 and ES6 JavaScript code with line counters,
+          Istanbul instruments your ES5 and ES2016+ JavaScript code with line counters,
```

Comment:
ES2015 is old and forgotten like ES3

On 3 Aug 2016 20:24, "Kent C. Dodds" <notifications@github.com> wrote:

> In index.html
> <https://github.com/istanbuljs/istanbuljs.github.io/pull/10#discussion_r73403302>
> :
>
> > @@ -14,7 +14,7 @@
> >
> >          <div class="page-heading">How Istanbul works</div>
> >          <p>
> > -          Istanbul instruments your ES5 and ES6 JavaScript code with line counters,
> > +          Istanbul instruments your ES5 and ES2016+ JavaScript code with line counters,
>
> Right, so this is effectively saying: "Istanbul instruments your ES5 and
> ES2016+, but doesn't do anything with ES2015 features"
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/istanbuljs/istanbuljs.github.io/pull/10/files/feaed766c653c92d8a451b8ef5e8e8c49b371024#r73403302>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAZQTGAta56ocNR0l3sLbIbINOn0mmijks5qcOrZgaJpZM4Jb-yd>
> .
>